docs/ispc.rst

```diff
@@ -1084,6 +1091,12 @@ preprocessor runs:
   * - ISPC_MINOR_VERSION
     - 13
     - Minor version of the ``ispc`` compiler/language
+  * - LLVM_VERSION_MAJOR
+    - 18
```

Comment:
Let's remove LLVM version number from the docs, we'll forget updating it for sure. Just keep the description.

Reply:
Same for ISPC_MAJOR_VERSION/ISPC_MINOR_VERSION.

Reply:
Done